components/datadog/apps/redis/ecs.go

```diff
@@ -0,0 +1,135 @@
+package redis
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
We should probably have something similar for the fakeintake, I'm thinking moving it out of scenarios. Not related to this PR